TwitchDownloaderWPF/PageChatRender.xaml.cs

```diff
@@ -43,10 +44,12 @@ private void btnBrowse_Click(object sender, RoutedEventArgs e)
         {
             OpenFileDialog openFileDialog = new OpenFileDialog();
             openFileDialog.Filter = "JSON Files | *.json;*.json.gz";
+            openFileDialog.Multiselect = true;
 
             if (openFileDialog.ShowDialog() == true)
             {
-                textJson.Text = openFileDialog.FileName;
+                fileNames = openFileDialog.FileNames;
+                textJson.Text = String.Join(",", fileNames);
```

Comment:
if the user edits textJson after using the open file dialog, their changes are not honored. This can be avoided by using a non-path valid character as the delimiter and adding a `TextChanged` event listener to reparse every change.

Reply:
Yeah I pretty much ignored the textbox entirely, only updated it to show that files were selected. Didn't even think about users trying to edit the path in the textbox itself.

Reply:
Yeah, it's unlikely it will happen but still possible.